python/ray/autoscaler/sdk.py

```diff
@@ -125,6 +130,8 @@ def rsync(cluster_config: Union[dict, str],
             public or private.
         no_config_cache (bool): Whether to disable the config cache and fully
             resolve all environment settings from the Cloud provider again.
+        all_nodes (bool): whether to sync worker nodes in addition to the head
```

Comment:
Didn't we want to deprecate this because of the race condition between rsync and starting up more workers?

Reply:
Can you explain more? I see we use it in the product and it seems like we do want to sync files to all the workers, what's the alternative to this?

Reply:
@ericl has previously expressed interest in keeping this private:
https://github.com/ray-project/ray/pull/11517#discussion_r509825458

Reply:
@ericl , is the goal to deprecate all_nodes in the `ray CLI`? If so, I can remove it as an option in the anyscale CLI

Reply:
Yes, we will remove this option.

Reply:
Thanks for catching this @pcmoritz ... somehow I missed it in first review.